nginx/nginx.tmpl

```diff
@@ -4,6 +4,9 @@ daemon off;
 error_log stderr {{ .LogLevel }};
 pid {{ .WorkingDir }}/nginx.pid;
 
+{{- $DefaultSslPath := .DefaultSslPath }}
+{{- $IngressPorts := .IngressPorts }}
```

Comment:
These are both used once directly, so is this aliasing necessary?

Reply:
Apparently inside a loop you lose the access to the variabe begining by "." so we need to set as template variable.

Reply:
Aliasing should be moved close to where it's used, rather than at the top of the file.